src/Microsoft.ML.Data/DataLoadSave/EstimatorExtensions.cs

```diff
@@ -18,7 +18,7 @@ public static class LearningPipelineExtensions
         /// Create a new composite reader estimator, by appending another estimator to the end of this data reader estimator.
         /// </summary>
         public static CompositeReaderEstimator<TSource, TTrans> Append<TSource, TTrans>(
```

Comment:
>CompositeReaderEstimator [](start = 22, length = 24)

So we now have a DataLoaderEstimator but now have a CompositeReaderEstimator. Should this also become a CompositeLoaderEstimator? And TrivialReaderEstimator->TrivialLoaderEstimator?

Reply:
Yes good point!

---
In reply to: [260396675](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260396675) [](ancestors = 260396675)